File: SalesLibraries.CloudAdmin/Business/Models/ExternalLibraryLinks/LibraryFolder.cs
namespace SalesLibraries.CloudAdmin.Business.Models.ExternalLibraryLinks
{
	class LibraryFolder
	{
		public string Id { get; set; }
		public string Name { get; set; }
		public int Order { get; set; }

		public LibraryLink[] Links { get; set; }
	}
}

File: WebSalesLibraries/protected/models/services_data/common/dictionaries/LibraryFolder.php
<?
	namespace application\models\services_data\common\dictionaries;

	/**
	 * Class LibraryFolder
	 */
	class LibraryFolder
	{
		public $id;
		public $name;
		public $order;

		/** @var  LibraryLink[] */
		public $links;
	}